src/test/java/com/github/stickerifier/stickerify/bot/StickerifyTest.java

```diff
@@ -19,6 +20,7 @@
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
 
+@ClearTempFiles
```

Comment:
Instead of an extension to clear our temp files, you could create instead one that sets the `'java.io.tmpdir'` system properties, and it could be used alongside the `@TempDir` extension (or it could extend it).

Reply:
I implemented a different way to clear temp files within [the last commit](https://github.com/Stickerifier/Stickerify/pull/134/commits/7644a3361debb03c4b38c57304713e3d850a3ed2) but I must say I prefer the previous one.

Please have a look at the changes, I'll revert the commit after that.